Formula/gmt.rb

```diff
@@ -60,7 +60,7 @@ def install
       system "cmake", "..", *args
       system "make", "install"
     end
-    inreplace bin/"gmt-config", "#{HOMEBREW_LIBRARY}/Homebrew/shims/mac/super/clang", "/usr/bin/clang"
+    inreplace bin/"gmt-config", Superenv.shims_path/ENV.cc, DevelopmentTools.locate(ENV.cc)
```

Comment:
```suggestion
    inreplace bin/"gmt-config", Superenv.shims_path/ENV.cc, ENV.cc
```

is that sufficient? It's what we do elsewhere

Reply:
It might be, but I was aiming for not changing the outcome of the `inreplace`, so that the replacement text is still, for example,`/usr/bin/clang` rather than just `clang`.

Reply:
Gotcha. There's maybe a few other places where we want to maintain this consistency 🤔